src/Allocs.jl

```diff
@@ -72,10 +72,10 @@ function pprof(alloc_profile::Profile.Allocs.AllocResults = Profile.Allocs.fetch
     samples = Vector{Sample}()
 
     sample_type = ValueType[
-        ValueType!("allocs", "count"), # Mandatory
-        ValueType!("size", "bytes")
+        ValueType!("alloc_objects", "count"), # Mandatory
+        ValueType!("alloc_space", "bytes")
     ]
-    period_type = ValueType!("heap", "bytes")
+    period_type = ValueType!("alloc_space", "bytes")
```

Comment:
oh maybe the period_type should actually be `(alloc_objects", "count")`? then maybe you could keep the period = 0x01?